certified-connectors/Mavim iMprove/apiProperties.json

```diff
@@ -0,0 +1,48 @@
+{
+    "properties": {
+      "connectionParameters": {
+        "token": {
+          "type": "oauthSetting",
+          "oAuthSettings": {
+            "identityProvider": "aad",
+            "clientId": "<<Please_add_your_clientID_here>>",
+            "redirectMode": "Global",
+            "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+            "properties": {
+              "IsFirstParty": "False",
+              "AzureActiveDirectoryResourceId": "{{Please_add_AzureActiveDirectoryResourceId_here}}"
+            },
+            "customParameters": {
+              "loginUri": {
+                "value": "https://login.windows.net"
+              },
+              "tenantId": {
+                "value": "common"
+              },
+              "resourceUri": {
+                "value": "<<Please_add_your_clientID_here>>"
```

Comment:
@RalphNoordanus The `resourceUri` should not be a client Id, Can you please provide exact value

Reply:
@sriyen-msft Added our Guid we use for this.